Lib/textwrap.py

```diff
@@ -429,7 +429,11 @@ def dedent(text):
     if not text:
         return text
```

Comment:
Actually... `textwrap.dedent(0)` would not return 0 =/ I hadn't thought of this... What do to? maybe still call `text.split('\n')` and check whether `lines == ['']` (in which case we do nothing)

Reply:
What do you think about [`d236d0c` (#132666)](https://github.com/python/cpython/pull/132666/commits/d236d0cf947b9433c94f8354aef303c48867b1a0)?

Reply:
Ah, we now have a warning:

> BytesWarning: Comparison between bytes and string

Reply:
I considered adding an explicit `isinstance`, but that slows down most ordinary calls, and doesn't feel very *Zen* -- functions operate on protocol and shape, so we should avoid being too restrictive.

Reply:
What about no checks at all and leave it to the `text.split()`? and for `''`, it'd be slightly slower. We're already faster than the previous implementation so...

Reply:
So you're suggesting removing the `text == ''` check? I don't think it hurts, but happy to do so if you'd prefer.

Reply:
Yes, I'm suggesting removing the fast path